source/MaterialXCore/UnitConverter.cpp

```diff
@@ -0,0 +1,208 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <algorithm>
+#include <MaterialXCore/Util.h>
+#include <MaterialXCore/Value.h>
+#include <MaterialXCore/UnitConverter.h>
+
+namespace MaterialX
+{
+const string DistanceUnitConverter::DISTANCE_UNIT = "distance";
+
+DistanceUnitConverter::DistanceUnitConverter(UnitTypeDefPtr unitTypeDef) :
```

Comment:
This class seems much more general than just distance units.  Would it be better named DefaultUnitConverter?

Reply:
We thought about this one for a bit and it can handle more than distance, but will only handle conversion where a scale is used. We didn't want to introduce an arbitrary offset just to make it a generic "scalar" unit converter. 

We could call it something else, but not sure what a good name would be, but it's not really a default as it would be a bit misleading. Any suggestions welcome.

Reply:
As mentioned by Ashwin, there is no mentioned or usage of offset.  DistanceUnitConverter is now called DefaultUnitConverter with a comment that it only handles conversion involving scalar multiplication.